src/Views/FormPages/PersonalInfo.jsx

```diff
@@ -105,9 +105,9 @@ function PersonalInfo(props){
             }}>
                 <div className="input-parent">
                     <Label className="text-bold" htmlFor="voter-contact" aria-hidden="true">
-                    {stringContent.emailLabel}<span className='required-text'>*</span>
+                    Voter Contact<span className='required-text'>*</span>
```

Comment:
Ray asked that this be moved from the json file and moved into the component file